package.json

```diff
@@ -4,32 +4,21 @@
   "description": "A water consumption app",
   "main": "index.jsx",
   "scripts": {
-    "dev": "webpack-dev-server --open --config ./webpack.config.js",
+    "dev": "webpack-dev-server --open --host 0.0.0.0 --config ./webpack.config.js",
     "build": "webpack --config webpack.config.js",
     "lint": "eslint **/*.{js,jsx} --quiet"
   },
   "repository": {
     "type": "git",
     "url": "git+https://github.com/FACN3/Water-you-talking-about.git"
   },
-  "keywords": [
-    "Water",
-    "Sea",
-    "Wet",
-    "Moist",
-    "H20"
-  ],
-  "author": [
-    "Amir ",
-    "Sami"
-  ],
+  "keywords": ["Water", "Sea", "Wet", "Moist", "H20"],
```

Comment:
make sure you both have the same configuration for auto formatting. There aren't any changes here just things on one line instead of two. It makes it harder to review so please change it thank you :upside_down_face:

Reply:
God damit prettier